Unix/http/httpauth.c

```diff
@@ -1529,9 +1538,12 @@ MI_Boolean IsClientAuthorized(_In_ Http_SR_SocketData * handler)
 
                 (* _g_gssState.Gss_Delete_Sec_Context)(&min_stat, &context_hdl, NULL);
 
-                handler->pAuthContext = NULL;
+                handler->pAuthContext  = NULL;
                 handler->authFailed = TRUE;
 
+                (* _g_gssState.Gss_Release_Cred)(&min_stat, handler->pVerifierCred);
```

Comment:
I'm checking it as good manners, but there is a trace message just above it.  pVerifierCred essentially couldn't be null here.